src/uct/ib/base/ib_md.c

```diff
@@ -1726,6 +1726,13 @@ uct_ib_md_open(const char *md_name, const uct_md_config_t *uct_md_config, uct_md
 void uct_ib_md_close(uct_md_h uct_md)
 {
     uct_ib_md_t *md = ucs_derived_of(uct_md, uct_ib_md_t);
+    uct_ib_md_open_entry_t *md_open_entry;
+
+    ucs_list_for_each(md_open_entry, &uct_ib_md_open_list, list) {
+        if (md_open_entry->md_cleanup) {
```

Comment:
would be nicer to point to ucs_empty_function instead of NULL (consistent with other funcptr-dtor code)